src/bitcoin-cli.cpp

```diff
@@ -867,6 +868,71 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) {
+        return;
+    }
```

Comment:
style nit, for a guard conditional at the top of the function this seems more compact to me, feel free to ignore
```diff
-    if (!find_value(result, "error").isNull()) {
-        return;
-    }
+    if (!find_value(result, "error").isNull()) return;
```

Reply:
Followed your suggestion and updated to a single line guard clause